### ISSUE TITLE
Osbuild-composer-10-1.fc32 - kernel NULL pointer dereference during compose

### ISSUE THREAD

**Issue by jrusz** [osbuild/osbuild-composer#525]
1) Provision a Fedora-32 vm
2) `dnf install osbuild-composer composer-cli`
3) Install https://bodhi.fedoraproject.org/updates/FEDORA-2020-c8975b7b3b and https://bodhi.fedoraproject.org/updates/FEDORA-2020-59a254a55a
4) create a file `base.toml` containing:
```
name = "base"
description = "A base system with bash v2"
version = "0.0.1"
modules = []
groups = []

[[packages]]
name = "bash"
version = "*"
```
5) `systemctl start osbuild-composer`
6) `composer-cli blueprints push base.toml`
7) `composer-cli compose start base ext4-filesystem`

The system freezes and the last thing I see on journal is:
```
Apr 22 06:06:37 hpe-dl320egen8-02.hpe2.lab.eng.bos.redhat.com kernel: EXT4-fs (loop0): mounted filesystem with ordered data mode. Opts: (null)
```
Looking at console I can see this:
```
[ 3014.126434] EXT4-fs (loop0): mounted filesystem with ordered data mode. Opts: (null)
[ 3032.508833] BUG: kernel NULL pointer dereference, address: 0000000000000000
[ 3032.546504] #PF: supervisor write access in kernel mode
[ 3032.573425] #PF: error_code(0x0002) - not-present page
[ 3032.596478] PGD 0 P4D 0 
[ 3032.607924] Oops: 0002 [#1] SMP PTI
[ 3032.623537] CPU: 2 PID: 23009 Comm: kworker/2:0 Tainted: G          I       5.6.2-301.fc32.x86_64 #1
[ 3032.664756] Hardware name: HP ProLiant DL320e Gen8 v2, BIOS P80 04/02/2015
[ 3032.695632] Workqueue: cgroup_destroy css_killed_work_fn
[ 3032.719572] RIP: 0010:bfq_bfqq_move+0x22/0x140
[ 3032.739474] Code: 0f 1f 84 00 00 00 00 00 0f 1f 44 00 00 41 56 41 55 49 89 d5 41 54 49 89 fc 55 48 89 f5 48 39 b7 a0 00 00 00 0f 84 08 01 00 00 <83> 45 00 01 48 89 ef e8 02 84 ff ff 85 c0 0f 85 bd 00 00 00 80 bd
[ 3032.824311] RSP: 0018:ffffad2d4c0e7dc8 EFLAGS: 00010086
[ 3032.848434] RAX: 0000000000000000 RBX: ffffa0d5ebf87120 RCX: ffffffffb33dd0d8
[ 3032.880726] RDX: ffffa0d5ec559800 RSI: 0000000000000000 RDI: ffffa0d5ec55a800
[ 3032.915263] RBP: 0000000000000000 R08: ffffa0d603042840 R09: 0000000000000000
[ 3032.948880] R10: 0000000000000000 R11: ffffa0d53e66b900 R12: ffffa0d5ec55a800
[ 3032.983354] R13: ffffa0d5ec559800 R14: ffffa0d5ec55abf0 R15: ffffa0d5ebf87098
[ 3033.016970] FS:  0000000000000000(0000) GS:ffffa0d606c80000(0000) knlGS:0000000000000000
[ 3033.056682] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
[ 3033.087116] CR2: 0000000000000000 CR3: 000000004c60a006 CR4: 00000000001606e0
[ 3033.120670] Call Trace:
[ 3033.132111]  bfq_pd_offline+0x87/0xf0
[ 3033.149372]  blkg_destroy+0x52/0xf0
[ 3033.165717]  blkcg_destroy_blkgs+0x4f/0xa0
[ 3033.184899]  css_killed_work_fn+0x4d/0xd0
[ 3033.203633]  process_one_work+0x1b4/0x380
[ 3033.222523]  worker_thread+0x53/0x3e0
[ 3033.239526]  ? process_one_work+0x380/0x380
[ 3033.259535]  kthread+0x115/0x140
[ 3033.274668]  ? __kthread_bind_mask+0x60/0x60
[ 3033.295100]  ret_from_fork+0x35/0x40
[ 3033.311942] Modules linked in: loop rfkill intel_rapl_msr intel_rapl_common ipmi_ssif x86_pkg_temp_thermal intel_powerclamp coretemp kvm_intel kvm irqbypass snd_pcsp snd_pcm crct10dif_pclmul crc32_pclmul snd_timer ghash_clmulni_intel ipmi_si iTCO_wdt snd gpio_ich intel_cstate ipmi_devintf iTCO_vendor_support intel_uncore soundcore tg3 hpilo hpwdt ipmi_msghandler intel_rapl_perf ie31200_edac acpi_power_meter lpc_ich ip_tables xfs libcrc32c mgag200 drm_kms_helper drm_vram_helper drm_ttm_helper ttm drm crc32c_intel serio_raw i2c_algo_bit
[ 3033.535808] CR2: 0000000000000000
[ 3033.551465] ---[ end trace 290cb3093acb263c ]---
[ 3033.577207] RIP: 0010:bfq_bfqq_move+0x22/0x140
[ 3033.600519] Code: 0f 1f 84 00 00 00 00 00 0f 1f 44 00 00 41 56 41 55 49 89 d5 41 54 49 89 fc 55 48 89 f5 48 39 b7 a0 00 00 00 0f 84 08 01 00 00 <83> 45 00 01 48 89 ef e8 02 84 ff ff 85 c0 0f 85 bd 00 00 00 80 bd
[ 3033.688846] RSP: 0018:ffffad2d4c0e7dc8 EFLAGS: 00010086
[ 3033.713484] RAX: 0000000000000000 RBX: ffffa0d5ebf87120 RCX: ffffffffb33dd0d8
[ 3033.747187] RDX: ffffa0d5ec559800 RSI: 0000000000000000 RDI: ffffa0d5ec55a800
[ 3033.780661] RBP: 0000000000000000 R08: ffffa0d603042840 R09: 0000000000000000
[ 3033.814322] R10: 0000000000000000 R11: ffffa0d53e66b900 R12: ffffa0d5ec55a800
[ 3033.849131] R13: ffffa0d5ec559800 R14: ffffa0d5ec55abf0 R15: ffffa0d5ebf87098
[ 3033.882776] FS:  0000000000000000(0000) GS:ffffa0d606c80000(0000) knlGS:0000000000000000
[ 3033.921036] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
[ 3033.948062] CR2: 0000000000000000 CR3: 000000004c60a006 CR4: 00000000001606e0
[-- MARK -- Wed Apr 22 10:10:00 2020 .. Wed Apr 22 10:20:00 2020 -- MARK --]
```
@jikortus helped me find this already existing bug and so I added it there as a reproducer https://bugzilla.redhat.com/show_bug.cgi?id=1822821

**Comment by teg** [comment]
Thanks for figuring this out! Closing as not our bug.

**Comment by ondrejbudai** [comment]
Should be fixed in kernel 5.6.7. This is really annoying. :disappointed:
